pkg/chartutil/values_test.go

```diff
@@ -350,6 +350,10 @@ func TestCoalesceValues(t *testing.T) {
 		}
 	}
 
+	// When the YAML value is null, we remove the value's key.
```

Comment:
- [ ] I apologize for not being more clear in [my original suggestion](https://github.com/kubernetes/helm/pull/2648#discussion_r126239691), but what I meant was that we should move only line 356 (starting with "ref:") over here to clarify why we look at these different ways of writing null in YAML, and leave the rest of the comment in place in the `coalesceValues` function.  

  I'd put this specification reference at [line 277](https://github.com/rbwsam/helm/blob/51b59497eed3537340c80750680e6626145520fc/pkg/chartutil/values_test.go#L277), just above the `testCoalesceValuesYaml` variable.